package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "heroku-pg",
   "description": "Heroku CLI plugin to manage Postgres.",
-  "version": "2.0.26",
+  "version": "2.0.27",
```

Comment:
it doesn't break any of the existing behaviour but it does add a lot of new commands so yes agreed, i'm updating now